source/js/mozfest-ga.js

```diff
@@ -0,0 +1,51 @@
+import ReactGA from "./react-ga-proxy.js";
+
+const bindMozfestGAEventTrackers = () => {
+  let homeWatchVideoButton = document.querySelector(
+    `#mozfest-home-watch-video-button`
+  );
+
+  if (homeWatchVideoButton) {
+    homeWatchVideoButton.addEventListener(`click`, () => {
+      ReactGA.event({
+        category: `CTA`,
+        action: `watch video tap`,
+        label: `watch video button tap`
+      });
+    });
+  }
+
+  let cmsPrimaryButtons = document.querySelectorAll(
+    `body.mozfest .cms a.btn.btn-primary`
+  );
+
+  if (cmsPrimaryButtons) {
+    cmsPrimaryButtons.forEach(button => {
+      button.addEventListener(`click`, () => {
+        ReactGA.event({
+          category: `CTA`,
+          action: `button tap`,
```

Comment:
@gideonthomas I used the word `tap` instead of `click` to follow our GA event action text convention